app/models/like.rb

```diff
@@ -1,4 +1,8 @@
 class Like < ApplicationRecord
-  belongs_to :user
+  belongs_to :author, class_name: 'User'
   belongs_to :post
+
+  def update_post_likes_counter
+    post.increment!(:likes_counter)
+  end
```

Comment:
- [SUGGESTION] We can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the `update_post_likes_counter` method every time a new post has been saved to the database, then make the `update_post_likes_counter` method private.

  Something like this:
  
  ```ruby
  # app/models/like.rb
    after_save :update_post_likes_counter
  
    private
  
    def update_post_likes_counter
      post.increment!(:likes_counter)
    end
  ```